### PR TITLE
Api caller

### PR DESCRIPTION
**Pull request by svbrodersen**
Api caller class and interface with simple unit tests for the two methods.

### CODE REVIEW

File: src/utility/OMDPCaller.java
Comment (rohanbrohan) [inline src/utility/OMDPCaller.java:1]: Slight typo with file name, should be OMDBCaller.java

**Review by rohanbrohan — CHANGES_REQUESTED** [review]
Looks good. Slight typo on file name for OMDBCaller but other than that I see no issues.

**Comment by svbrodersen** [comment]
fixed the typo

**Review by Alex732004 — COMMENTED** [review]
Api caller looks good

**Review by Alex732004 — APPROVED** [review]

**Review by Alex732004 — APPROVED** [review]
Typo is fixed

**Review by rohanbrohan — APPROVED** [review]

**Pull request merged**